include/dsn/tool-api/network.h

```diff
@@ -304,6 +304,7 @@ class rpc_session : public ref_counter
     void clear_send_queue(bool resend_msgs);
     bool on_disconnected(bool is_write);
     bool is_auth_success(message_ex *msg);
+    bool in_security_white_list(task_code code);
```

Comment:
```
bool rpc_session::in_security_white_list(task_code code)
{
    return security::is_negotiation_message(code) || fd::is_failure_detector_message(code);
}
```

This is totally static. Why make it a method rather than a free function?

```
namespace security {
bool in_white_list(task_code code)
{
    return is_negotiation_message(code) || fd::is_failure_detector_message(code);
}
} // 
```